ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py

```diff
@@ -291,21 +292,35 @@ def test_jax_nn_one_hot(
         min_num_dims=1,
     ),
     axis=helpers.ints(min_value=-1, max_value=0),
```

Comment:
there is a `dtype_values_axis` helper to generate dtype, x and its valid axis together (by toggling force_int_axis and valid_axis to True), might be an improvement to data generation

Reply:
Thanks!